pkg/enforce/enforce.go

```diff
@@ -66,6 +67,10 @@ func EnforceAll(ctx context.Context, ghc *ghclients.GHClients) error {
 		}
 		opt.Page = resp.NextPage
 	}
+	log.Info().
+		Str("area", "bot").
+		Int("count", len(insts)).
```

Comment:
should we use more prescriptive names e.g. installationCount, 
similar below one, like repoCount ?
and something specific for id, like XYZID

Reply:
I feel like it would be too verbose to have a unique name for each tag-meaning across all logs, but maybe not, not sure.
I've been mostly querying based on the message name, then the tag name is essentially scoped to that message.

Reply:
fine as-is, was a nit suggestion. those names are nice if used as stackdriver metrics. i think here it is just a log message, so fine either way.